app/src/main/res/xml/preferences.xml

```diff
@@ -103,6 +103,14 @@
             android:summaryOn="@string/max_refresh_rate_enabled"
             app:key="max_refresh_rate"
             app:title="@string/max_refresh_rate" />
+        <emu.skyline.preference.IntegerListPreference
+            android:defaultValue="0"
+            android:entries="@array/aspect_ratios"
+            android:entryValues="@array/aspect_ratios_val"
+            app:key="aspect_ratio"
+            app:refreshRequired="true"
```

Comment:
I don't think a game list refresh is needed when this setting is changed so you might as well remove this line too. Though I agree the name of the attribute is confusing and that's on me.